core/server/master/src/main/java/alluxio/master/file/meta/InodeTree.java

```diff
@@ -849,6 +853,11 @@ public InodeDirectory getRoot() {
           mDirectoryIdGenerator.getNewDirectoryId(rpcContext.getJournalContext()),
           currentInodeDirectory.getId(), name, directoryContext);
 
+      if (directoryContext.getXAttr() != null) {
+        LOG.info("[czhu] CreateDirectoryContext createPath::setXAttr {}", directoryContext.getXAttr());
+        newDir.setXAttr(directoryContext.getXAttr());
+      }
+
```

Comment:
See [this comment](https://github.com/Alluxio/alluxio/pull/15205/files#r835513332) about feature completeness for setting `xAttr`.